WordPress/src/main/java/org/wordpress/android/ui/reader/repository/ReaderPostRepository.kt

```diff
@@ -29,16 +35,18 @@ class ReaderPostRepository(
     private val shouldAutoUpdateTagUseCase: ShouldAutoUpdateTagUseCase,
     private val getPostsForTagWithCountUseCase: GetPostsForTagWithCountUseCase,
     private val fetchPostsForTagUseCase: FetchPostsForTagUseCase,
-    private val readerUpdatePostsEndedHandler: ReaderUpdatePostsEndedHandler
+    private val readerUpdatePostsEndedHandler: ReaderUpdatePostsEndedHandler,
+    private val postLikeActionUseCase: PostLikeActionUseCase
 ) : CoroutineScope {
     override val coroutineContext: CoroutineContext
         get() = bgDispatcher
 
     private var isStarted = false
+    private var isDirty = false
 
-    private val _postsForTag = ReactiveMutableLiveData<ReaderPostList>(
-            onActive = { onActivePostsForTag() }, onInactive = { onInactivePostsForTag() })
-    val postsForTag: LiveData<ReaderPostList> = _postsForTag
+    private val _mutablePosts = ReactiveMutableLiveData<ReaderPostList>(
```

Comment:
In general, we don't use the `mutable` prefix in the codebase. The underscore is what indicates that the livedata are mutable. I'd consider removing the `mutable` part. Wdyt?

Reply:
No biggie, I'll remove it.